### DISCUSSION TITLE
How to set up streaming?

### DISCUSSION THREAD

**Discussion by Morrowj4** [langflow-ai/langflow#431] [Q&A]
I saw the announcement about now supporting streaming, but can anyone give me some pointers about the steps?

The LLM modules don't seem to have any settings for it. Is there something else I need to add?

**Comment by majumba** [comment]
did you manage to find a solution in the end?

**Comment by evolu8** [comment]
I've flagged streaming to on but nothing. Seems to be rendering at a good speed. But not streaming in the chat.

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.

**Reply by Yalab7** [reply]
After trying the latest version of LangFlow, I still couldn't find the option to enable streaming and I couldn't find it in the docs either. Can you point me to the steps to enable it?

**Comment by carlosrcoelho** [comment]
As an example, you can take the OpenAI LLM, go to the Advanced option of the component, and select Stream.

<img width="1945" alt="image" src="https://github.com/user-attachments/assets/64ad910d-9d24-4176-b20b-d307aae0449a">

**Reply by wuhy198607** [reply]
but when I deployed locally,  selected the option, but still no stream response.
